app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -324,29 +327,63 @@ fun CreateActivityScreen(
-                expanded = expanded,
-                onExpandedChange = { expanded = !expanded }) {
+                expanded = expandedType,
+                onExpandedChange = { expandedType = !expandedType }) {
                   OutlinedTextField(
                       readOnly = true,
-                      value = selectedOption,
+                      value = selectedOptionType,
                       onValueChange = {},
                       label = { Text("Activity Type") },
                       trailingIcon = {
-                        ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded)
+                        ExposedDropdownMenuDefaults.TrailingIcon(expanded = expandedType)
                       },
                       colors = ExposedDropdownMenuDefaults.textFieldColors(),
-                      modifier = Modifier.menuAnchor().fillMaxWidth())
+                      modifier = Modifier.menuAnchor().fillMaxWidth().testTag("typeTextField"))
                   ExposedDropdownMenu(
-                      expanded = expanded,
-                      onDismissRequest = { expanded = false },
+                      expanded = expandedType,
+                      onDismissRequest = { expandedType = false },
                       modifier = Modifier.fillMaxWidth().padding(STANDARD_PADDING.dp)) {
                         types.forEach { selectionOption ->
                           DropdownMenuItem(
                               modifier = Modifier.fillMaxWidth().padding(STANDARD_PADDING.dp),
                               text = { Text(selectionOption.name) },
                               onClick = {
-                                selectedOption = selectionOption.name
-                                expanded = false
+                                selectedOptionType = selectionOption.name
+                                expandedType = false
+                              })
+                        }
+                      }
+                }
+
+            ExposedDropdownMenuBox(
+                modifier =
+                    Modifier.testTag("chooseCategoryMenu")
+                        .align(Alignment.CenterHorizontally)
+                        .fillMaxWidth()
+                        .padding(STANDARD_PADDING.dp),
+                expanded = expandedCategory,
+                onExpandedChange = { expandedCategory = !expandedCategory }) {
+                  OutlinedTextField(
+                      readOnly = true,
+                      value = selectedOptionCategory,
+                      onValueChange = {},
+                      label = { Text("Activity Category") },
+                      trailingIcon = {
+                        ExposedDropdownMenuDefaults.TrailingIcon(expanded = expandedCategory)
+                      },
+                      colors = ExposedDropdownMenuDefaults.textFieldColors(),
+                      modifier = Modifier.menuAnchor().fillMaxWidth().testTag("categoryTextField"))
```

Comment:
We will most definitely unify colors and UI conventions in the next sprints ! But in the meantime, I ran it locally and it looks nice.